src/cascadia/TerminalApp/TabManagement.cpp

```diff
@@ -888,6 +888,34 @@ namespace winrt::TerminalApp::implementation
             co_await _HandleCloseTabRequested(tab);
         }
     }
+    // Method Description:
+    // - Responds to changes in the TabView's item list by changing the
+    //   tabview's visibility.
+    // - This method is also invoked when tabs are dragged / dropped as part of
+    //   tab reordering and this method hands that case as well in concert with
+    //   TabDragStarting and TabDragCompleted handlers that are set up in
+    //   TerminalPage::Create()
+    // Arguments:
+    // - sender: the control that originated this event
+    // - eventArgs: the event's constituent arguments
+    void TerminalPage::_OnTabItemsChanged(const IInspectable& /*sender*/, const Windows::Foundation::Collections::IVectorChangedEventArgs& eventArgs)

```

Comment:
I think we should really start using an observable item collection instead of manipulating the tab view list directly. It might many of the animation bugs we have too...